Wikipedia/Code/WMFExploreSectionControllerCache.m

```diff
@@ -52,9 +47,7 @@ - (instancetype)initWithDataStore:(MWKDataStore*)dataStore {
     if (self) {
         self.dataStore                              = dataStore;
         self.sectionControllersBySection            = [[NSMutableDictionary alloc] init];
-        self.reverseLookup                          =
-            [NSMapTable mapTableWithKeyOptions:NSMapTableWeakMemory | NSMapTableObjectPointerPersonality
-                                  valueOptions:NSMapTableWeakMemory];
+        self.reverseLookup                          = [[NSMutableDictionary alloc] init];
```

Comment:
Never mind… you are now using the hash… so that we do't have to deal with the section controllers themselves